test-utils/state-viewer/src/state_dump.rs

```diff
@@ -1,25 +1,25 @@
-use std::collections::HashMap;
-use std::fs;
-use std::fs::File;
-use std::path::Path;
-
-use serde::ser::{SerializeSeq, Serializer};
-
 use near_chain::RuntimeAdapter;
 use near_chain_configs::Genesis;
+use near_crypto::PublicKey;
+use near_primitives::account::id::AccountId;
 use near_primitives::block::BlockHeader;
 use near_primitives::state_record::StateRecord;
-use near_primitives::types::{AccountInfo, StateRoot};
+use near_primitives::types::{AccountInfo, Balance, StateRoot};
 use near_store::TrieIterator;
 use nearcore::config::NearConfig;
 use nearcore::NightshadeRuntime;
+use serde::ser::{SerializeSeq, Serializer};
+use std::collections::HashMap;
+use std::fs;
+use std::fs::File;
+use std::path::Path;
 
 pub fn state_dump(
```

Comment:
comment what this function does - especially the case where records_path is present vs missing